_includes/links.md

```diff
@@ -10,6 +10,7 @@
 [cran-checkpoint]: https://cran.r-project.org/package=checkpoint
 [cran-knitr]: https://cran.r-project.org/package=knitr
 [cran-stringr]: https://cran.r-project.org/package=stringr
+[curriculum-handbook]: https://carpentries.github.io/curriculum-development/
```

Comment:
I've just noticed that you added `curriculum-handbook` link here -- please remove it: changes to this file have to be made in carpentries/styles repo.

Reply:
Thank you @maxim-belkin. I added the link directly to `index.md` to prevent a "Line too long" warning.